apps/nextra/pages/en/build/smart-contracts/smart-table.mdx

```diff
@@ -0,0 +1,192 @@
+---
+title: "Smart Table"
+---
+
+# Smart Table
+
+The Smart Table is a scalable hash table implementation based on linear hashing. 
+This data structure aims to optimize storage and performance by utilizing linear hashing, which splits one bucket at a time instead of doubling the number of buckets, thus avoiding unexpected gas costs. 
+The Smart Table uses the SipHash function for faster hash computations while tolerating collisions.
```

Comment:
This is a different description from `Table`, but it's similar.  Need more information on tradeoffs